Python/Include/ListTrial.hpp

```diff
@@ -0,0 +1,48 @@
+#ifndef ROVER_PYTHON_LIST_TRIAL_HPP
+#define ROVER_PYTHON_LIST_TRIAL_HPP
+#include <string_view>
+#include <pybind11/pybind11.h>
+#include "Rover/ListTrial.hpp"
+#include "Rover/Sample.hpp"
+
+namespace Rover {
+  
+  //! Exports a Trial for Python object samples
+  /*!
+    \param module The module to export the class to
+  */
+  void export_list_trial(pybind11::module& module);
+
+  //! Exports a Trial for a particular Sample type
+  /*!
+    \param module The module to export the class to
+    \param postfix The postfix to the name of the exported type
+  */
+  template<typename S>
+  void export_list_trial(pybind11::module& module, std::string_view postfix);
+
+  template<typename S>
+  void export_list_trial(pybind11::module& module, std::string_view postfix) {
+    auto name = std::string("Constant").append(postfix);
+    pybind11::class_<ListTrial<S>>(module, name.c_str())
+      .def(pybind11::init<>())
+      .def("reserve", &ListTrial<S>::reserve)
+      .def("capacity", &ListTrial<S>::capacity)
+      .def("insert", (void (ListTrial<S>::*)(const S&))
+         &ListTrial<S>::insert)
+      .def("insert", [](ListTrial<S>& out,
```

Comment:
Since I cannot properly export iterator-based insert, nor is it conventional in Python, I replaced it with a ListTrial insert. Let me know if I should substitute it with something else.